src/backend/sgx/data.rs

```diff
@@ -154,16 +156,19 @@ pub fn epc_size(max: u32) -> Datum {
 }
 
 pub fn dev_sgx_enclave() -> Datum {
-    let mut pass = false;
-
-    if File::open("/dev/sgx_enclave").is_ok() {
-        pass = true;
-    }
-
     Datum {
         name: "Driver".into(),
-        pass,
+        pass: File::open("/dev/sgx_enclave").is_ok(),
         info: Some("/dev/sgx_enclave".into()),
         mesg: None,
     }
 }
+
+pub fn aesm_socket() -> Datum {
+    Datum {
+        name: "AESM Daemon Socket".into(),
+        pass: Path::new(AESM_SOCKET).exists(),
```

Comment:
Why `File::open` for `/dev/sgx_enclave` and just checking that path exists here?
Shouldn't we have consistent way to check these? (probably `open` should be preferred?)

Reply:
right

Reply:
true

Reply:
well... you can't open it :) you can only connect() to it :)